src/utils/componentUtilityFunctions.js

```diff
@@ -2,6 +2,7 @@ import delay from 'lodash/delay';
 import moment from 'moment';
 import { sortStates } from 'carbon-components-react/lib/components/DataTable/state/sorting';
 import fileDownload from 'js-file-download';
+import isNil from 'lodash/isNil';
```

Comment:
Fun fact, just found this: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator